personalPortfolio/portfolio.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nosipho Manyase</title>
+    <link href = "styles.css" rel="stylesheet" > 
+</head>
+<body>
+  <div class = "first-section">
+  <p class = "my-name">Hi, I'm Nosipho! </p>
```

Comment:
I think it would be cool to add a button somewhere that links to your LinkedIn and Github profile